projects/plugins/social/src/js/index.js

```diff
@@ -3,11 +3,15 @@
  */
 import React from 'react';
 import ReactDOM from 'react-dom';
```

Comment:
Yeah, I'm happy to. This is also based on [the starter plugin](https://github.com/Automattic/jetpack/blob/master/projects/plugins/starter-plugin/src/js/index.js#L4), it might be worth checking in with somebody else to see if there's a specific reason the `@wordpress/*` stuff isn't being used here.

Reply:
Yeah I think it's something to consider in the future, but I think we should swap to the `@wordpress/` packages. We can have that discussion elsewhere